task.py

```diff
@@ -75,7 +76,7 @@ def create_node_port_service(self, nodeport: int) -> str:
         logger.info(f"Creating Node Port Service {out_file_yaml}")
         r = self.run_oc(f"apply -f {out_file_yaml}")
         if r.returncode != 0:
-            if "already exists" not in r.err:
+            if "already exists" not in r.err and self.reuse_node:
```

Comment:
Should this be reuse pod? Maybe just override this function in your task rather than needing to add another switch to the generic Task ABC